packages/kbn-rison/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@kbn/rison",
+  "private": true,
+  "version": "1.0.0",
+  "main": "./target_node/index.js",
```

Comment:
should there be a `browser` entry point?

Reply:
There could be, but I'm going to be removing all those in the next couple days 😉